src/Polytopes/PolyhedralFan/properties.jl

```diff
@@ -76,14 +76,43 @@ function Base.iterate(iter::MaximalConeIterator, index = 1)
 end
 Base.length(iter::MaximalConeIterator) = nmaximal_cones(iter.PF)
 
-# function cones(as::Type{T}, PF::PolyhedralFan, cone_dim::Int) where {T}
-#     rtype = AsTypeIdentitiesC(as)
-#     if (cone_dim < 0)
-#         return nothing
-#     end
-#     rcones = Polymake.fan.cones_of_dim(PF.pm_fan,cone_dim-length(lineality_space(PF)))
-#     return PolyhedronOrConeIterator{rtype}(PF.pm_fan.RAYS,rcones, PF.pm_fan.LINEALITY_SPACE)
-# end
+@doc Markdown.doc"""
+    cones(as::Type{T} = Cone, PF::PolyhedralFan, cone_dim::Int)
+
+Return the cones of `PF` of dimension `cone_dim` as an iterator over the type of
+object given by `as`.
+
+Optional arguments for `as` include
+* `Cone`.
```

Comment:
So, what types other than `Cone` could one pass? ... and how would one find out, if the natural place to find out (here!) doesn't tell us? is there a general guiding principle perhaps?

```suggestion
The parameter `as` can be used to control the type of objects the iterator produces.
If omitted, it defaults to `Cone`.
```

Reply:
I am not sure what possible return types are planned so I keep this possibility open for now. If we do not want to use it, we can easily remove it at a later time. Again, this is similar to `faces(::Polyhedron)`.

Reply:
I don't know... it strikes me as weird to add and document an option with only a single allowed argument just for the sake of it... if `faces` does the same, it should be removed in `faces`, too?